css-layout-classes/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+
+<head>
+  <title>CSS layout classes</title>
+  <link href="styles.css" rel="stylesheet">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <h1>Choose your character!</h1>
+    </div>
+    <div class="row">
+      <div class="column-third">
+        <img src="images/link.png">
+      </div>
+      <div class="column-third">
+        <h2>Super Smash Bros. Ultimate</h2>
+        <p>Super SmashBros. Ultimate received universal acclaim from both criticis and players, with some critics calling it the best installment in the series.It received praise for its large amount of content and fine-tuning of existing Smash gameplay elements, although its online mode was widely critized.</p>
+      </div>
+      <div class="column-third">
+        <img src="images/yoshi.png">
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-fourth">
+        <img src="images/kirby.png">
+      </div>
+      <div class="column-fourth">
+        <img src="images/fox.png">
+      </div>
+      <div class="column-fourth">
+        <img src="images/pikachu.png">
+      </div>
+      <div class="column-fourth">
+        <img src="images/ness.png">
+      </div>
+    </div>
+    <div class="row">
+      <div class="column">
+        <h2>Donkey Kong</h2>
+        <p>Donkey Kong is a series of video games featuring the adventures of a gorilla called Donkey Kong, conceived by Shigeru Miyamoto in 1981. The franchise consists mainly of two game genres, but also includes additional spin-off titles of various genres.</p>
+        <p>The games of the first genre are most single-screen plattform/action puzzle types, featuring Donkey Kong as the opponent in an industrial construction setting.</p>
+      </div>
+      <div class="column">
+        <img src="images/dk.png">
+      </div>
+    </div>
+    <div class="row">
+      <div class="column">
+        <img src="images/mario.png">
+      </div>
+      <div class="column">
+        <h2>Mario</h2>
+        <p>Mario is fictional character in the Mario video game franchise, owned by Nintendo and created by Japanese video game designer Shigeru Miyamoto. Acting as the company's mascot as well as being the eponymous protagonist of the series, Mario has apppeared in over 200 video games since his creation.</p>
+        <p>Depicted as a short, pudgy, Italian plumber who resides in the Mushroom Kingdom, his adventures generally center upon rescuing Princess Peach from the Koopa villain Bowser.</p>
+      </div>
+    </div>
```

Comment:
Looks like your `column` classes got a bit lazy here 👀 

Please create a `column` class with an appropriate name and styling for the size these portions are meant to take up